ark/phenotyping/som_utils.py

```diff
@@ -1935,3 +1935,58 @@ def generate_weighted_channel_avg_heatmap(cell_cluster_channel_avg_path, cell_cl
         bbox_transform=plt.gcf().transFigure,
         loc='upper right'
     )
+
+
+def add_consensus_labels_cell_table(base_dir, cell_table_name, cell_consensus_name):
+    """Adds the consensus cluster labels to the cell table,
+    then resaves data to `base_dir/cell_table_name`
+
+
+    Args:
+        base_dir (str):
```

Comment:
I think this is a good place to tackle the cell table location issue as well:
https://github.com/angelolab/ark-analysis/issues/523